cddl/mec-endorsement-triple-record.cddl

```diff
@@ -0,0 +1,4 @@
+multi-env-conditional-endorsement-triple-record = [
+  conds: [ + condition-triple-record ]
+  actions: [ + stateful-environment-record ]
```

Comment:
IIRC we agreed this would be the other way round, i.e.: the condition expressed in terms of a bunch of stateful-environments, the "actions" as EMTs.

Reply:
Maybe. Let's fix that nit in the next call.